README.md

```diff
@@ -525,6 +525,28 @@ config.view_component.preview_route = "/previews"
 
 This example will make the previews available from <http://localhost:3000/previews>.
 
+#### Adding contextual HTML for previews
+Given the preview component lives in `test/components/previews/cell_component_preview.rb`, template files can be defined at `test/components/previews/cell_component_preview/`.
+
+`test/components/previews/cell_component_preview.rb`
+```ruby
+class CellComponentPreview < ViewComponent::Preview
+  def default
```

Comment:
Rails controllers allow templates to be rendered without defining a method in the controller. Might we consider supporting similar behavior, not requiring `#default` to be defined in this case where a template is also provided?

Reply:
That will involve changing the concept of an "example" in the Preview object. Currently one example is just one method in the preview object. The example might call render in its body (initial previewing approach) or have an empty body and use a template (new approach).

But we definitely can support this. We would just need to update the preview object.

Reply:
Well I went a bit different about this, just with the intention to keep the implementation simple. I introduced a specific method to render a preview with a template and make the preview object have all the examples defined in there as methods, as we currently do.